src/Support/Testing/Fakes/ConsumerBuilderFake.php

```diff
@@ -0,0 +1,448 @@
+<?php
+
+namespace Junges\Kafka\Support\Testing\Fakes;
+
+use Closure;
+use InvalidArgumentException;
+use Junges\Kafka\Config\Sasl;
+use Junges\Kafka\Config\Config;
+use Junges\Kafka\Support\Timer;
+use Junges\Kafka\Config\BatchConfig;
+use Junges\Kafka\Config\NullBatchConfig;
+use Junges\Kafka\Consumers\CallableConsumer;
+use Junges\Kafka\Contracts\MessageDeserializer;
+use Junges\Kafka\Commit\Contracts\CommitterFactory;
+use Junges\Kafka\Exceptions\KafkaConsumerException;
+use Junges\Kafka\Support\Testing\Fakes\ConsumerFake;
+use Junges\Kafka\Contracts\HandlesBatchConfiguration;
+
+class ConsumerBuilderFake
```

Comment:
Can we extend this from `Junges\Kafka\Consumers\ConsumerBuilder` and change only the methods that needs a different behavior? I notice that this class is pretty much the same as the ConsumerBuilder.